account_banking_pain_base/models/account_payment_order.py

```diff
@@ -515,6 +515,11 @@ def generate_party_acc_number(
             party_account_other = etree.SubElement(party_account_id, "Othr")
             party_account_other_id = etree.SubElement(party_account_other, "Id")
             party_account_other_id.text = partner_bank.sanitized_acc_number
+        if self.payment_type == "outbound" and party_type == "Dbtr":
+            currency = etree.SubElement(party_account, "Ccy")
+            currency.text = (
+                self.journal_id.currency_id.name or self.company_currency_id.name
+            )
```

Comment:
In Italy with this node get invalid payment file from bank, i think checking the pain version should be added in `if` condition and the node should be added only for international pain flavour

Reply:
hi @luc-demeyer this the bank error about Ccy attribute
![image](https://github.com/user-attachments/assets/205816db-545f-4e11-a917-85880d342ce6)
You can find the xsd for the italian sct on https://github.com/OCA/l10n-italy/blob/16.0/l10n_it_sct_cbi/data/standards/CBIPaymentRequest.00.04.01.xsd#L45
If you check the type CBICashAccount2 there is no mention about Ccy node

The sct file is modified to hide some information, but the nodes are untouched, the extension is txt because Github not allow to load xml file, need to change the fiel extension
[sct_PAY0318_OCA.txt](https://github.com/user-attachments/files/20406109/sct_PAY0318_OCA.txt)